packages/rest/src/__tests__/acceptance/routing/routing.acceptance.ts

```diff
@@ -135,6 +135,48 @@ describe('Routing', () => {
     );
   });
 
+  it('allows controllers to define single value array params via decorators', async () => {
+    class MyController {
+      @get('/greet')
+      greet(
+        @param.array('stringArray', 'query', {type: 'string'})
+        stringArray: string[],
+      ) {
+        return `hello ${stringArray}`;
+      }
+    }
+    const app = givenAnApplication();
+    const server = await givenAServer(app);
+    givenControllerInApp(app, MyController);
+    return (
```

Comment:
@mrmodise why is it required to return in the test? I would expect that it should be enough to test with `expect`

Reply:
I found this to be the standard across all tests in routing. I didnt want to move away from the existing pattern

Reply:
This is weird. I think we should `await` the response returned by `whenIMakeRequestTo` instead.

```ts
await whenIMakeRequestTo(server)
  // ...
```

I am ok to keep the current style for consistency with existing tests.

Reply:
Thanks @bajtos